src/components/ItemLookup.vue

```diff
@@ -26,6 +30,11 @@ const emit = defineEmits( {
 } );
 
 const searchInput = ref( '' );
+watchEffect( () => {
```

Comment:
Not as far as I can tell, if I remove this block then the input stays empty (and the Lookup doesn’t open the options menu either when clicking into it).

Reply:
I took this pattern from Query Builder, by the way: https://gerrit.wikimedia.org/g/wikidata/query-builder/+/37545c11ff9246f80264ac816101c3ee1c3b3909/src/components/EntityLookup.vue#105

(But with `watchEffect` because it’s not clear to me how to `watch` a prop in the composition API… I think `watch( toRefs( props ).value` would be the alternative?)

Reply:
Ah right. Looking at the Wikit LookupInput code, it seems that it updates that label itself only when actually _selecting_ an option, but not when one is given to it.

Reply:
I agree that `watchEffect()` it probably best here